packages/cloudflare/src/cli/config.ts

```diff
@@ -1,103 +1,5 @@
-    source: {
-      // Path to the next application
-      root: string;
-      // Path to the app's `.next` directory (where `next build` saves the build output)
-      dotNext: string;
-      // Path to the application standalone root directory
-      standaloneRoot: string;
-    };
-    output: {
-      // Path to the output directory
-      root: string;
-      // Path to the OpenNext static assets directory
-      assets: string;
-      // Path to the app's `.next` directory in the OpenNext output directory
-      dotNext: string;
-      // Path to the application standalone root directory
-      standaloneRoot: string;
-      // Path to the application standalone directory (where `next build` saves the standalone app)
-      standaloneApp: string;
-      // Path to the `.next` directory specific to the standalone application
-      standaloneAppDotNext: string;
-      // Path to the `server` directory specific to the standalone application
-      standaloneAppServer: string;
-    };
-    internal: {
-      // Package in the standalone node_modules
-      package: string;
-      // Templates in the package in the standalone node_modules
-      templates: string;
-    };
-  };
-
-  // Internal name for the copy of the package
-  internalPackageName: string;
-};
-
-/**
- * Computes the configuration.
- *
- * @param projectOpts The options for the project
- * @returns The configuration, see `Config`
- */
-export function getConfig(projectOpts: ProjectOptions): Config {
-  const sourceDirDotNext = join(projectOpts.sourceDir, ".next");
-
-  const dotNext = join(projectOpts.outputDir, ".next");
-  const appPath = getNextjsApplicationPath(dotNext).replace(/\/$/, "");
-  const standaloneRoot = join(dotNext, "standalone");
-  const standaloneApp = join(standaloneRoot, appPath);
-  const standaloneAppDotNext = join(standaloneApp, ".next");
-  const standaloneAppServer = join(standaloneAppDotNext, "server");
-
-  const nodeModules = join(standaloneApp, "node_modules");
-  const internalPackage = join(nodeModules, ...PACKAGE_NAME.split("/"));
-  const internalTemplates = join(internalPackage, "cli/templates");
-
-  return {
-    build: {
-      skipNextBuild: projectOpts.skipNextBuild,
-      shouldMinify: projectOpts.minify,
-    },
-
-    paths: {
-      source: {
-        root: projectOpts.sourceDir,
-        dotNext: sourceDirDotNext,
-        standaloneRoot: join(sourceDirDotNext, "standalone"),
-      },
-      output: {
-        root: projectOpts.outputDir,
-        assets: join(projectOpts.outputDir, "assets"),
-        dotNext,
-        standaloneRoot,
-        standaloneApp,
-        standaloneAppDotNext,
-        standaloneAppServer,
-      },
-      internal: {
-        package: internalPackage,
-        templates: internalTemplates,
-      },
-    },
-
-    internalPackageName: PACKAGE_NAME,
-  };
-}
+import { statSync } from "node:fs";
+import { join } from "node:path";
 
 export function containsDotNextDir(folder: string): boolean {
```

Comment:
not full related to your refactoring, but I think that this function shouldn't be here, but it should be a local non-exported function in `/src/cli/build/build.ts` instead

Reply:
I'll actually drop that, ON has `checkRunningInsideNextjsApp` already